.github/workflows/scheduled-ci.yml

```diff
@@ -0,0 +1,124 @@
+name: Scheduled CI
```

Comment:
All of these items can stay in small runners as they run on a cron and are not time sensitive.